resources/config.yaml

```diff
@@ -66,6 +66,17 @@ docker:
     image:    # set by docker.py based on framework name, lowercase
     tag:      # set by docker.py based on framework version
 
+singularity:
+  force_branch: true  # set to true if docker image can only be built from a clean branch, wth same tag as defined in `project_repository`.
+  minimize_instances: true
+  library: 'francisco.rivera.valverde/default'
```

Comment:
please don't use your private library as default.
You can always override this locally in your `~/.config/automlbenchmark/config.yaml` file.